### PR TITLE
feat added: list and input component added

### PR DESCRIPTION
**Pull request by Tanjum786**
**Added list**
_created different types of lists like-_

- unordered list
- ordered list
- stacked list 
- inline list
- without bullet list

**Added input to the component **
inputs like-

- input filed 
- input filed with validation
Screenshot from 2022-02-09 13-15-30

### CODE REVIEW

File: component/input/input.css [hunk 1]
```diff
@@ -0,0 +1,28 @@
+div.input_box {
```
Comment (tanishq20) [inline component/input/input.css]: Avoid giving styles like this. It makes hard specificity chain.

File: index.html [hunk 2]
```diff
@@ -401,9 +405,206 @@ <h2 class="component_example">Example For Simple Ratings</h2>
...
+                <h2 class="component_example">Example simple input filed</h2>
+                <div>
+                    <p class="para">
+                        Inputs are used in forms to collect user related data.
+                    </p>
+                </div>
+                <div class="component_box input_box">
+                    <div>
+                        <label for="username" class="lable_text">Username *</label>
+                        <input type="text" name="text" class="user_text" placeholder="Enter your name">
```
Comment (RahulSingh9131) [inline index.html]: add Id="username" in Input , this specifies that both label and input are connected.

File: component/list/list.css [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+
+.list_notify {
+    display: flex;
+    justify-content: space-between;
+    width: 20rem;
+    padding: 1rem;
+    border-radius: 0.5rem;
+}
+
+.notify1 {
```
Comment (tanishq20) [inline component/list/list.css]: You can better class name for better readability.

**Review by tanishq20 — COMMENTED** [review]
Suggested few things.

File: component/input/input.css (hunk 1 above)
Comment (RahulSingh9131) [inline component/input/input.css]: are u using here div for specificity, if not then you can remove this.

**Review by RahulSingh9131 — APPROVED** [review]
Add id attribute in each of your input tag. Else everything looks good to me.

File: index.html (hunk 2 above)
Comment (Tanjum786) [reply index.html]: ohh ok i wil change it

File: component/input/input.css (hunk 1 above)
Comment (Tanjum786) [reply component/input/input.css]: okay

Comment (Tanjum786) [reply component/input/input.css]: yes for specificity only it's bad practice I think I will correct it

**Review by tanishq20 — APPROVED** [review]

**Pull request merged**